### ISSUE TITLE
overpass API features

### ISSUE THREAD

**Issue by harry-wood** [harry-wood/xapi-ui#15]
could add support for overpass API style querying. e.g. a tab for the query XML

That's something of a duplication with this of course: http://overpass-turbo.eu   ...but then overpass turbo doesn't fill in bbox values.

**Comment by mmd-osm** [comment]
As XAPI is basically dead, I would even go as far and replace the current XAPI tab by an "Overpass QL" tab. But then the name "XAPI Query builder" becomes sort of moot. 😎

**Comment by harry-wood** [comment]
Indeed. What would be a good new name for a tool like this?  ..."tag bboxer" or something

**Comment by mmd-osm** [comment]
I’m pretty much lacking any talent to come up with good names but I could offer some help to get current xapi queries migrated to ql.
